packages/service-library/requirements/_aiohttp.in

```diff
@@ -5,14 +5,12 @@
 
 --constraint ./_base.in
 
-openapi-core==0.12.0     # frozen until https://github.com/ITISFoundation/osparc-simcore/pull/1396 is CLOSED
-lazy-object-proxy~=1.4.3 # cannot upgrade due to contraints in openapi-core
-aiozipkin
-
 aiohttp
 aiopg[sa]
+aiozipkin
+attrs
```

Comment:
Q: why do we need attrs?

Reply:
there are several modules in ``serviceslib.aiohttp``  still using it. I could replace them by @classmodel but in a separate PR.